scripts/azure-linux_mac-release.yml

```diff
@@ -105,21 +105,6 @@ steps:
   displayName: 'Build libtiledb'
   continueOnError: true
 
-- task: ArchiveFiles@2
```

Comment:
There is two separate uploads. This one was the entire source tree+build dir. The second upload is the release artifacts.